tests/MongoDB.Driver.Core.Tests/Specifications/initial-dns-seedlist-discovery/tests/longer-parent-in-return.json

```diff
@@ -10,7 +10,7 @@
   ],
   "options": {
     "replicaSet": "repl0",
-    "ssl": true
+    "tls": true
```

Comment:
We don't want to be different than the spec repo. We want to be in sync with it.

Unless the spec repo is changed let's keep this at `ssl` and just make it work.

Same comment for any other changes like this one.

Reply:
I have no problem with updating the spec tests...

...but, they must be updated in the specifications repository, not in ours.

We want to keep our copy of the files in sync with the specifications repository.

Reply:
Yes, I meant updating tests in spec repo, not only in ours. But probably the current solution also looks good enough. Thanks.